esmvaltool/cmor/table.py

```diff
@@ -25,18 +25,28 @@ def read_cmor_tables(cfg_developer):
         Parsed config-developer file
 
     """
+    custom = CustomInfo()
+    CMOR_TABLES['custom'] = custom
+
     for table in cfg_developer:
         project = cfg_developer[table]
 
         table_path = project.get('cmor_tables', '')
```

Comment:
It would probably be good to have esmvaltool/cmor/tables/<project_name> as the default path instead.

Reply:
I changed the code to make it a bit clearer